hack/build-cross.sh

```diff
@@ -20,6 +25,8 @@ OS_BUILD_PLATFORMS=("${platforms[@]}")
 host_platform=$(os::build::host_platform)
 if [[ $host_platform == "linux/ppc64le" ]]; then
   OS_GOFLAGS_LINUX_PPC64LE="-tags=gssapi" os::build::build_binaries "${OS_CROSS_COMPILE_TARGETS[@]}"
+elif [[ $host_platform == "linux/arm64" ]]; then
```

Comment:
Oh, of course. We're not in a function so `local` doesn't make sense. That sounds good.